peachjam/adapters/indigo.py

```diff
@@ -735,6 +740,13 @@ def get_edit_url(self, document):
         if self.settings.get("indigo_url"):
             return f"{self.settings['indigo_url']}/works{document.work_frbr_uri}"
 
+    def is_stranded_expression(self, document):
```

Comment:
Wouldn't this include stub expressions without any PIT dates? eg. https://edit.laws.africa/works/akn/za/act/p/1994/r86/